pom.xml

```diff
@@ -40,6 +40,8 @@
     <spotless.check.skip>false</spotless.check.skip>
     <ban-junit4-imports.skip>false</ban-junit4-imports.skip>
     <hpi.strictBundledArtifacts>true</hpi.strictBundledArtifacts>
+    <hpi.bundledArtifacts>jsr305</hpi.bundledArtifacts>
+    <spotbugs.skip>true</spotbugs.skip>
```

Comment:
We never want to introduce spotbugs.skip on new code because this ruleset is set up to keep the most problematic code out.

Reply:
The `<spotbugs.skip>true</spotbugs.skip>` property is defined on line 43. Remove it. If the build is failing with it removed, fix the problem it is reporting. If the problem cannot be fixed, use `@SuppressWarnings` on that block of code rather than globally ignoring.